query-input/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>de.brandwatch.minianalytics</groupId>
+    <artifactId>mini-analytics</artifactId>
+    <version>0.1</version>
+    <!-- lookup parent from repository -->
+  </parent>
+
+  <artifactId>query-input</artifactId>
```

Comment:
I'd still suggest to rename this, as it's gonna be much more

Reply:
Done